py4DSTEM/io/datastructure/py4dstem/braggvectors_fns.py

```diff
@@ -345,6 +346,19 @@ def index_bragg_directions(
 
     self.braggdirections = braggdirections
 
+    if self.calibration.get_Q_pixel_units() == 'A^-1':
```

Comment:
I like that we are taking the hardline atomic physicist approach, that the data isn't calibrated unless it's calibrated in Ångström units. SI is for _chemists_